### DISCUSSION TITLE
部分构建(build)markdown文件如何配置

### DISCUSSION THREAD

**Discussion by raysun0501** [vuepress-theme-hope/vuepress-theme-hope#3315] [Q&A] [answered]
场景描述：
一次新建了很多篇博客(markdown)，但是只写完了其中1篇，也只想发布到网站上一篇，其他未完成的博客不希望发布到网站上，请问如何配置才可以构建(build)指定的博客(markdown文件)到dist目录？

**Comment by Mister-Hope** [comment, marked as answer]
[pagePatterns](https://v2.vuepress.vuejs.org/reference/config.html#pagepatterns)
